icu4c/source/test/intltest/loctest.cpp

```diff
@@ -5592,17 +5592,12 @@ class FavorScriptLocale : public Locale {
     }
 };
 
-
 bool isKnownSourceForCLDR17099(const std::string& s) {
     if (s.compare("qaa-Cyrl-CH") == 0) {
         return true;
     }
-    if (s.compare(0, 9, "und-Latn-") != 0) {
-        return false;
-    }
-    std::string tail(s.substr(9));
-    std::string regionsForCLDR17099("AE CC ER HK IL IN MV PK RS SD SS");
-    return (regionsForCLDR17099.find(tail) != std::string::npos);
+    
+	return false;
```

Comment:
care to fix the indentation? or too much trouble right now?

Reply:
I'll take care of that in next iteration. Otherwise I would have to re-merge source changes which will kick off the whole check set again :(

Reply:
Hopefully in the next iteration this logKnownIssue test skip will go away since the underlying issue will get fixed. Then the indentation is moot...

Reply:
@DraganBesevic Hmm, the problem with marking this a logKnownIssue skip for CLDR-17099 is that (1) CLDR-17099 does not mention the qaa-Cyrl-CH issue (and yes, I think I meant to add that to it earlier), but more importantly, (2) CLDR-17099 is marked as done and fixed, so having a logKnownIssue skip for it is odd.

I thin what we need to do as a minimum is clone CLDR-17099 to a new ticket for the qaa-Cyrl-CH issue, and then in CLDR-17099 mention that there was actually one more issue for qaa-Cyrl-CH, and mention the clone ticket that tracks it. Otherwise we will lose track of this.